src/pages/Search.js

```diff
@@ -1,31 +1,52 @@
 import React, { useState, useEffect } from 'react';
 import { useParams, withRouter, useHistory } from 'react-router-dom';
 import SubredditForm from '../components/SubredditForm';
+import Spinner from '../components/Spinner';
+import Heatmap from '../components/Heatmap';
+import PostTable from '../components/PostTable';
+import useFetch from '../hooks/useFetch';
 
 
 const Search = () => {
   const { subreddit: subredditQuery } = useParams();
   const history = useHistory();
   const [subreddit, setSubreddit] = useState(subredditQuery);
+  const [url, setUrl] = useState(`https://www.reddit.com/r/${subreddit}`);
```

Comment:
Here you're basically duplicating the `subreddit` state. You can remove this state and use a simple const instead

```
const url = `https://www.reddit.com/r/${subreddit}`;
const [response, loading, hasError] = useFetch(url);
```

or 

```
const [response, loading, hasError] = useFetch(`https://www.reddit.com/r/${subreddit}`);
```

But as I said above I'd actually just pass the `subreddit` to the `useFetch` hook and build the URL there